### PR TITLE
Add Token standard. Add Transaction missing fields.

### PR DESCRIPTION
**Pull request by kolya182**

### CODE REVIEW

File: blockatlas/transaction-format.md [hunk 1]
```diff
@@ -13,7 +29,10 @@
    "to": "123",
    "fee": "1234"
    "coin": 60,
-   "block": 1
+   "date": 1562256431,
+   "block": 123,
+   "memo": "",
+   "status": ""
```
Comment (vikmeup) [inline blockatlas/transaction-format.md]: missing default `status` value

File: blockatlas/transaction-format.md [hunk 2]
```diff
@@ -168,3 +187,27 @@ will continue... Keys mostly used to provide localized version on the clients by
...
+
+## Token
+
+## Base
+
+`name` - Token name
+
+`symbol` - Token symbol
+
+`decimals` - Number of decimla places
```
Comment (vikmeup) [inline blockatlas/transaction-format.md]: Misspelled `decimal`

Comment (kolya182) [reply blockatlas/transaction-format.md]: https://github.com/TrustWallet/developer/pull/31/commits/ff8246bd37e9629b96b48e2d2b537994d4bbf86e

File: blockatlas/transaction-format.md (hunk 1 above)
Comment (kolya182) [reply blockatlas/transaction-format.md]: https://github.com/TrustWallet/developer/pull/31/commits/ff8246bd37e9629b96b48e2d2b537994d4bbf86e

**Pull request merged**